sycl/include/CL/sycl/detail/pi.h

```diff
@@ -285,7 +285,9 @@ typedef enum {
   PI_DEVICE_INFO_GPU_SLICES = 0x10023,
   PI_DEVICE_INFO_GPU_SUBSLICES_PER_SLICE = 0x10024,
   PI_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE = 0x10025,
-  PI_DEVICE_INFO_MAX_MEM_BANDWIDTH = 0x10026
+  PI_DEVICE_INFO_MAX_MEM_BANDWIDTH = 0x10026,
+  // These are extensions that are currently only implemented for nvidia.
```

Comment:
```suggestion
  // These are extensions that are currently implemented for CUDA plugin only.
```
Not sure it's a good idea to separate/group extensions supported by a single plug-in, so I'd like hear from the runtime team.

Reply:
Since this is standard SYCL, we should not tell here which plugins implement it, just remove the comment?

Reply:
Sounds good to me. I'll remove the comment.